assignment/scripts/aboutMe.js

```diff
@@ -1,59 +1,91 @@
-
+let food = "Burrito";
 // 9 - Create a variable called `pets` and set it to the value of the number of pets you have
-
+let pets = 1;
 // 10 - Create a variable called `friendsPets` and assign it the value of the number of pets your friend has
-
+let friendsPets = 2;
 // 11 - Add two pets to your `pets` variable
-
+let allPets = friendsPets + pets;
 // 12 - Create a constant variable called `allowedPets` and set it to a number value of your choice
-
-// 13 - Create a conditional: if adventurous is true, console log "Adventures are great!", 
+const allowedPets = 3;
+// 13 - Create a conditional: if adventurous is true, console log "Adventures are great!",
 // if it's not true, console log "How about we stay home?"
-
-
+adventures ? console.log("Adventures are great!") : console.log("How about we stay at home ? ");
 // 14 - Create a compound conditional: if luckyNumber is 2 and adventurous is true,
 // console log "Roll the dice!"
-
-// 15 - Write a conditional that console logs "I can have more pets!" 
+if (luckyNumber == 2 && adventures) {
+  console.log("Roll the dice!");
+}
+// 15 - Write a conditional that console logs "I can have more pets!"
 //if the value of `pets` is less than the value of `allowedPets`,
 // console logs "I have enough pets" if the value of `pets` is equal to the value of `allowedPets`,
 // and console logs "Oh no, I have too many pets!"
 // if the value of `pets` is greater than the value of `allowedPets`.
-
+if (pets < allowedPets) {
+  console.log("I can have more pets!");
+} else if (pets === allowedPets) {
+  console.log("I have enough pets");
+} else {
+  // another condition is not necessary here IMO, since there can be no other option
+  console.log("I have too many pets!");
+}
 // STRETCH GOALS:
 
 // 16 - Make a variable called `mostPets` and a conditional that
 // correctly checks the `pets` and `friendsPets` variables, and
 // assigns the highest value to `mostPets`. There's several possibilities --
-// be sure to think through all the scenarios. 
+// be sure to think through all the scenarios.
 // console.log `mostPets` after the conditional has run.
 
+let mostPets;
+
+if (pets > friendsPets) {
+  mostPets = pets;
+  console.log("I have the most pets", mostPets);
+} else if (pets === friendsPets) {
+  console.log("We have the same amount of pets", mostPets);
+} else {
+  mostPets = friendsPets;
+  console.log("My friends have the most pets", mostPets);
+}
+
 // 17 - Write a *switch* statement that logs:
 //      "First is the worst" if your lucky number is 1
 //      "Second is the best" if your lucky number is 2
 //      "Third is the one with the polka dot dress" if your lucky number is 3
 //      Otherwise, log "Luck is what happens when preparation meets opportunity"
 //      You'll need to research how to use switch statements!
 
-// 18 -- Rewrite question 13 with a `ternary` operator. You'll need to do research!
-
+switch (luckyNumber) {
```

Comment:
Take another look at the docs for [switch statements](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch). What you have here will produce some weird behavior. For example, if your luck number were 2, this would log out:

```
Second is the best
Third is the one with the polka dot dress
Luck is what happens when preparation meets opportunity
```

Can you figure out why and how to fix it?

Reply:
**I forgot the break statements!**